spacecmd/src/spacecmd/ssm.py

```diff
@@ -92,9 +92,8 @@ def do_ssm_add(self, args):
         if system in self.ssm:
             logging.warning(_N('%s is already in the list') % system)
             continue
-        else:
-            self.ssm[system] = self.get_system_id(system)
-            logging.debug('Added %s' % system)
+        self.ssm[system] = self.get_system_id(system)
+        logging.debug('Added %s' % system)
 
     if self.ssm:
         logging.debug('Systems Selected: %i' % len(self.ssm))
```

Comment:
Same here

Reply:
@agraul , `logging.debug('Added %s', system)` causes the unit test to fail.

Reply:
Seems that we're comparing log messages in code and tests, that's not really nice. I'd say just keep it as it is then, if you don't get pylint complains everything is fine.